server/etcdserver/api/rafthttp/peer.go

```diff
@@ -250,30 +250,16 @@ func (p *peer) send(m raftpb.Message) {
 		if isMsgSnap(m) {
 			p.r.ReportSnapshot(m.To, raft.SnapshotFailure)
 		}
-		if p.status.isActive() {
-			if p.lg != nil {
-				p.lg.Warn(
-					"dropped internal Raft message since sending buffer is full (overloaded network)",
-					zap.String("message-type", m.Type.String()),
-					zap.String("local-member-id", p.localID.String()),
-					zap.String("from", types.ID(m.From).String()),
-					zap.String("remote-peer-id", p.id.String()),
-					zap.String("remote-peer-name", name),
-					zap.Bool("remote-peer-active", p.status.isActive()),
-				)
-			}
-		} else {
-			if p.lg != nil {
-				p.lg.Warn(
-					"dropped internal Raft message since sending buffer is full (overloaded network)",
-					zap.String("message-type", m.Type.String()),
-					zap.String("local-member-id", p.localID.String()),
-					zap.String("from", types.ID(m.From).String()),
-					zap.String("remote-peer-id", p.id.String()),
-					zap.String("remote-peer-name", name),
-					zap.Bool("remote-peer-active", p.status.isActive()),
-				)
-			}
+		if p.lg != nil {
+			p.lg.Warn(
+				"dropped internal Raft message since sending buffer is full",
```

Comment:
> Additionally, the unnecessary context (in parentheses) in the log
message was removed, which was necessary without the zap context (with
the old logger), but now only confuses.

In what way does it confuse now, can you maybe elaborate? Thanks!

Reply:
Because this "context" was the same in case of (possible) network overload [L256](https://github.com/etcd-io/etcd/pull/13238/files/daf7e0350a77ae2866ef193c8c64c4328d281d02#diff-7b682c3417ecefeb3283bcf2cc9da682a307771fd455a6b81dd3d909339aa50eL256) (or possible network failure) and when the remote peer was unavailable [L268](https://github.com/etcd-io/etcd/pull/13238/files/daf7e0350a77ae2866ef193c8c64c4328d281d02#diff-7b682c3417ecefeb3283bcf2cc9da682a307771fd455a6b81dd3d909339aa50eL268)
This context is added based on assumptions about the problem that occurred on the network and may be inaccurate (or not true at all)

This context (assumption) in parentheses was useful when it was impossible to add a zap context
[rafthttp/peer.go#L230](https://github.com/etcd-io/etcd/blob/fdbedacc8347f0a5ff13efe077c46300b29de545/rafthttp/peer.go#L230)

In my case, I break the network channel between two nodes of the cluster, and for some time (while the unavailable node is considered active), a message about network overload appears in the log, but there is no overload, there is a complete network break.
It seems to me that the zap context [R261](https://github.com/etcd-io/etcd/pull/13238/files/daf7e0350a77ae2866ef193c8c64c4328d281d02#diff-7b682c3417ecefeb3283bcf2cc9da682a307771fd455a6b81dd3d909339aa50eR261) is enough to understand what is happening.